docs/en/notebooks/ECCO.ipynb

```diff
@@ -1155,9 +1155,8 @@
    "id": "b67a4bbd-91d1-4aac-b23d-ea8a2d7cd9ae",
    "metadata": {},
    "source": [
-    "```{tip}\n",
-    "You can use the `consolidated_metadata method` to rapidly download all metadata associated with a datacube, and cache it.\n",
-    "This is a feature of pydap >= 3.5.5.\n",
+    "```{warning}\n",
+    "For `Pydap >= 3.5.5` the new method `consolidated_metadata` can be used to download and cache all dmrs in parallel. This can significantly speed up the (xarray) dataset exploration. THis method continues under active development and so its features will continue to evolve.\n",
```

Comment:
Minor typo: THis --> This. But maybe That last sentence could be: "However, this method remains under active..."